honeybee_schema/energy/constructionset.py

```diff
@@ -229,5 +375,60 @@ class ConstructionSetAbridged(NamedEnergyBaseModel):
     )
 
 
+class ConstructionSet(ConstructionSetAbridged):
+    """A set of constructions for different surface types and boundary conditions."""
+
+    type: constr(regex='^ConstructionSet$') = 'ConstructionSet'
+
+    wall_set: WallSet = Field(
+        default=None,
+        description='A WallSet object for this ConstructionSet.'
+    )
+
+    floor_set: FloorSet = Field(
+        default=None,
+        description='A FloorSet object for this ConstructionSet.'
+    )
+
+    roof_ceiling_set: RoofCeilingSet = Field(
+        default=None,
+        description='A RoofCeilingSet object for this ConstructionSet.'
+    )
+
+    aperture_set: ApertureSet = Field(
+        default=None,
+        description='A ApertureSet object for this ConstructionSet.'
+    )
+
+    door_set: DoorSet = Field(
+        default=None,
+        description='A DoorSet object for this ConstructionSet.'
+    )
+
+    materials: List[
+        Union[
+            EnergyMaterial, EnergyMaterialNoMass,
+            EnergyWindowMaterialGas, EnergyWindowMaterialGasMixture,
+            EnergyWindowMaterialGasCustom, EnergyWindowMaterialBlind,
+            EnergyWindowMaterialGlazing, EnergyWindowMaterialShade,
+            EnergyWindowMaterialSimpleGlazSys]
+        ] = Field(
+                ...,
+                description='List of materials. The order of the materials is from outside '
+                    'to inside.',
+                min_items=1,
+                max_items=8
+            )
+
+    layers: List[constr(min_length=1, max_length=100)] = Field(
```

Comment:
This key isn't a part of the non-abridged `ConstructionSet`. Maybe you meant to put `constructions` here.

Reply:
Yes, this is a mistake.